src/jsx.d.ts

```diff
@@ -1263,19 +1263,24 @@ export namespace JSXInternal {
 		/** @deprecated See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contextmenu */
 		contextmenu?: Signalish<string | undefined>;
 		controls?: Signalish<boolean | undefined>;
-		controlsList?: Signalish<string | undefined>;
+		controlslist?: Signalish<'nodownload' | 'nofullscreen' | 'noremoteplayback' | undefined>;
+		controlsList?: Signalish<'nodownload' | 'nofullscreen' | 'noremoteplayback' | undefined>;
```

Comment:
Might be worth altering `AllHTMLElements` to get all of its properties from `IntrinsicElements`, eliminating the need to copy from the more per-element interfaces. Will give that a go separately